docs/developers-guide-drivers.md

```diff
@@ -1,12 +1,12 @@
-# Metabase Driver Development Guide
+# Community-built Database Drivers
 
 As of Metabase 0.32, we now support community-built database drivers!
 
 Several folks have already started to build drivers for database integrations that Metabase doesn't natively support. While we don't specifically endorse any of these drivers, we thought it would be helpful to have a central place to list them.
 
 Want to build your own driver? Take a look at the [driver development](#driver-development) section below.
 
-## Community Database Drivers
+## How to use a community-built driver
```

Comment:
All of your other headings and subheadings use Title Case — pick a side, man! 😂 

FWIW, my personal preference is for `Sentence case` for headings and subheadings because I think it looks less formal. Our existing docs aren't perfectly consistent, but they primarily tend to use sentence case (because I wrote a whole lot of them :b ). 

It's probably time for me to bang out a draft of a docs/copy-writing style guide to collaborate on with you and @kdoh (he might have even started a draft like a year ago).

Reply:
🤣 I just want to give every heading equal opportunity! 

I'll go through and make them more consistent. 🙏